tests/src/Simple/MultiModule/MultiModule.cs

```diff
@@ -35,4 +38,23 @@ public static int TestStaticBases()
         
         return Pass;
     }
+
+    public static int TestSharedGenerics()
+    {
+        // Use a generic dictionary that also exists in the library
```

Comment:
Nit: I like the growing pattern we have of each sub test case printing out what it's testing to the log